.travis.yml

```diff
@@ -0,0 +1,36 @@
+---
+language: node_js
+node_js:
+  # Using the same minimum supported node version as Ember CLI & Lerna
+  - '6'
+
+cache:
+  directories:
+    - $HOME/.npm
+    - $HOME/packages/*/.npm
+    - $HOME/packages/*/.cache # includes bowers cache
+
+env:
+  - PACKAGE=navi-app
```

Comment:
Can we put this under `env` --> `matrix`

https://docs.travis-ci.com/user/environment-variables/#Global-Variables

Reply:
Isn't it by default a `matrix` env var?